### PR TITLE
Add a few timeline tests that includes select/un-select stats vars

### PR DESCRIPTION
**Pull request by wh1210**

### CODE REVIEW

File: server/webdriver_tests/timeline_test.py [hunk 1]
```diff
@@ -12,11 +12,16 @@
...
 # limitations under the License.
 
+
 import unittest
 import urllib
 from base_test import WebdriverBaseTest
-
+ 
 TIMELINE_URL = '/tools/timeline'
+STATVAR_TEST_URL_1 = '#&statsVar=Median_Age_Person,0,1__Median_Income_Person,0,2__Count_Person_Upto5Years,'\
```
Comment (shifucun) [inline server/webdriver_tests/timeline_test.py:21]: can you rename this to "URL_HASH_1"?

File: server/webdriver_tests/timeline_test.py [hunk 2]
```diff
@@ -40,10 +45,54 @@ def test_charts_original(self):
         """Test the original timeline page. No charts in this page."""
         self.driver.get(self.url_ + TIMELINE_URL)
         self.driver.implicitly_wait(5)
-        responses = []
-        responses = self.driver.find_elements_by_class_name("card")
-        self.assertEqual(len(responses), 0)
+        charts = []
```
Comment (shifucun) [inline server/webdriver_tests/timeline_test.py:48]: remove this line?

File: server/webdriver_tests/timeline_test.py [hunk 3]
```diff
@@ -40,10 +45,54 @@ def test_charts_original(self):
...
+        self.assertEqual(len(charts), 0)
 
+    def test_charts_from_url_directly_and_uncheck_statvar(self):
+        """
+        Given the url directly, test the menu and charts are shown correctly.
+        Then unclick one statvar, test the corresponding change.
+        """
+        self.driver.get(self.url_ + TIMELINE_URL + STATVAR_TEST_URL_1)
+        self.driver.implicitly_wait(5)
+        charts = []
```
Comment (shifucun) [inline server/webdriver_tests/timeline_test.py:59]: can remove this and the ones below?

Comment (wh1210) [reply server/webdriver_tests/timeline_test.py]: Done

File: server/webdriver_tests/timeline_test.py (hunk 2 above)
Comment (wh1210) [reply server/webdriver_tests/timeline_test.py]: Done

File: server/webdriver_tests/timeline_test.py (hunk 1 above)
Comment (wh1210) [reply server/webdriver_tests/timeline_test.py]: Done

**Pull request merged**